app/src/main/java/com/yacgroup/yacguide/database/Country.kt

```diff
@@ -18,9 +18,9 @@
 package com.yacgroup.yacguide.database
 
 import androidx.room.Entity
+import androidx.room.PrimaryKey
 
-@Entity(primaryKeys = ["name"])
-class Country {
-
-    var name = ""
-}
+@Entity
+data class Country(
```

Comment:
FYI: `data` classes allow property-wise object comparison by implementing an according `==` operator. I use this feature in the adapter to check the difference of list items (which are updated by RecyclerView if they differ). Example: Returning to the route activity while its ascent bitmask has changed (and it hence should be colored).